samples/readme.md

```diff
@@ -0,0 +1,120 @@
+# Plug-in Creation for Performance SDK
+
+This document will introduce two paths for creating Performance Toolkit SDK (SDK) plug-ins:
+
+1) Simple Data Source
+2) Extensible Data Source
+
+## Overview
```

Comment:
As someone who doesn't understand either of these sources, this overview didn't help me understand any differences between them. Can we add, broad definitions of what they are? That would make the comparisons more meaningful.